### PR TITLE
Search Console Datapoint Refactoring

### PR DESCRIPTION
**Pull request by aaemnnosttv**
## Summary

Addresses issue #449

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by aaemnnosttv** [comment]
e2e tests passing locally 👍

File: includes/Modules/Search_Console.php [hunk 1]
```diff
@@ -460,6 +456,15 @@ protected function setup_info() {
 		);
 	}
 
+	/**
+	 * Get the configured Webmasters service instance.
+	 *
+	 * @return Google_Service|\Google_Service_Webmasters
+	 */
+	private function get_webmasters_service() {
```
Comment (felixarntz) [inline includes/Modules/Search_Console.php]: Same concern like on #492, I'd prefer to drop this method and use a class constant for the service (e.g. `SERVICE_WEBMASTERS`).

Comment (felixarntz) [reply includes/Modules/Search_Console.php]: @aaemnnosttv Can you please update this similar to my comment in https://github.com/google/site-kit-wp/pull/492#discussion_r321742874?

Comment (aaemnnosttv) [reply includes/Modules/Search_Console.php]: Done!

**Pull request merged**